tests/cpp/test_gpu_view.cpp

```diff
@@ -368,33 +367,54 @@ std::vector<ReshapeExample> reshape_after_reduce_examples = {
     {{1, 27454, 1, 2}, {1, 3922, 1, 7}},
     {{1, 7844, 1, 7}, {1, 1961, 4}}};
 
-namespace {
-using ReshapeBeforeReduction = NVFuserFixtureParamTest<ReshapeExample>;
-TEST_P(ReshapeBeforeReduction, FusionReshapeBeforeReduction) {
-  const auto& [input_shape, output_shape] = GetParam();
-  maybeClearAllocator(); // Shmoo tests can occupy a lot of memory
-  reductionViewAddFusion(
-      input_shape, output_shape, /*reshape_before_reduction=*/true);
+namespace ReshapeReduction {
```

Comment:
```suggestion
namespace {
```

Is there a specific reason you gave it a name? An anonymous namespace looks just fine in a cpp file.

Reply:
just in case there are multiple anonymous namespaces with same named vars, we have test files close to 10K lines of code.